Managers/SceneMan.h

```diff
@@ -420,21 +419,27 @@ class SceneMan : public Singleton<SceneMan>, public Serializable {
     unsigned char GetTerrMatter(int pixelX, int pixelY);
 
 
-//////////////////////////////////////////////////////////////////////////////////////////
-// Method:          GetMOIDPixel
-//////////////////////////////////////////////////////////////////////////////////////////
-// Description:     Gets a MOID from pixel coordinates in the Scene. LockScene() must be
-//                  called before using this method.
-// Arguments:       The X and Y coordinates of screen Scene pixel to get the MO from.
-// Return value:    The MOID currently at the specified pixel location.
-
+	/// <summary>
+	/// Gets a MOID from pixel coordinates in the Scene. LockScene() must be called before using this method.
+	/// </summary>
+	/// <param name="pixelX">The X coordinate of the Scene pixel to test.</param>
+	/// <param name="pixelY">The Y coordinate of the Scene pixel to test.</param>
+	/// <param name="ignoreTeam">The team to ignore.</param>
+	/// <returns>The MOID currently at the specified pixel coordinates.</returns>
     MOID GetMOIDPixel(int pixelX, int pixelY, int ignoreTeam);
+
+    /// <summary>
+    /// Gets a MOID from pixel coordinates in the Scene. LockScene() must be called before using this method.
+    /// </summary>
+    /// <param name="pixelX">The X coordinate of the Scene pixel to test.</param>
+    /// <param name="pixelY">The Y coordinate of the Scene pixel to test.</param>
+    /// <returns>The MOID currently at the specified pixel coordinates.</returns>
     MOID GetMOIDPixel(int pixelX, int pixelY) { return GetMOIDPixel(pixelX, pixelY, Activity::NoTeam); }
 
     /// <summary>
-    /// Gets this scene's MOID spatial partitioning grid
+    /// Gets this Scene's MOID SpatialPartitioningGrid.
```

Comment:
There’s no such thing as a SpatialPartitioningGrid. Rename to SpatialPartitionGrid.

Reply:
Oops!